contributor-guide/modules/ROOT/pages/getting-involved.adoc

```diff
@@ -1,4 +1,58 @@
 = Getting Involved
 :navtitle: Getting Involved
 
-*Content under construction.*
\ No newline at end of file
+There are many ways to get involved with Boost, up to and including submitting your own library. In roughly ascending order of time and work commitment, here are the ways you can contribute to Boost:
+
+== 1. Join the Boost Users Mailing List
+
+The Boost mailing list is a platform for discussions and communications related to the Boost Libraries. The mailing list serves several key purposes:
+
+[disc]
```

Comment:
Unless you want to be explicit about the bullet type, I think Spencer already fixed this.

Reply:
Indeed, removed [disc]